src/adapter/src/coord/dataflows.rs

```diff
@@ -97,11 +97,38 @@ impl Coordinator {
 
     /// Finalizes a list of dataflows and then broadcasts it to all workers.
     async fn ship_dataflows(&mut self, dataflows: Vec<DataflowDesc>, instance: ComputeInstanceId) {
```

Comment:
Encouragement to swap the names here! The one that doesn't panic should be named `ship_dataflows` and return a `Result`, then the other should be called `must_ship_dataflow` or something and it documents that it panics on error.

Or, this should document that it panics.

Reply:
Agreed, done.